src/wallet/wallet.cpp

```diff
@@ -175,7 +175,7 @@ CPubKey CWallet::GetKeyFromKeypath(uint32_t nChange, uint32_t nChild) {
     return pubkey;
 }
 
-CPubKey CWallet::GenerateNewKey(uint32_t nChange)
+CPubKey CWallet::GenerateNewKey(uint32_t nChange, bool nWriteChain)
```

Comment:
Why do you start boolean variables with `n`? In bitcoin it's `f` to denote variable as a "flag"